Various technical notes/transition_infos.json

```
@@ -329,7 +324,7 @@
         },
         {
           "area_id": "0xECC9D759",
-          "area_name": "Flooded Courtyard (2 exits)",
+          "area_name": "Flooded Courtyard",
```

dark caverns and scorpions

```suggestion
          "area_name": "Flooded Courtyard (2 exits)",
```

---

I explicitly removed all those '(2 exits)' descriptions.
If you really, really want those back I would like to hear your reason as to why.

---

Information.

We took the time to identify which transitions we still have no theorically way to even handle. It could be made a comment instead, but then either we need implementations to support `JSONC` (which honestly, why not, Internet Explorer is dead, og json can die too please, most things support json with comments and trailing commas nowadays) (note, no need to go full `JSON5`)

Or we add all that information to the comment entry: `"//" :`

---

I suggest I don't touch this any further for now.
I'll leave you to add these descriptions back in as comments once you've found a way of doing so.
But that's for a future release, not this one.